lib/tool_shed/test/base/populators.py

```diff
@@ -31,12 +34,40 @@
 )
 from .api_util import ShedApiInteractor
 
+try:
```

Comment:
You could import this from galaxy.util.resources

Reply:
Ahhh - yeah - good call! I've fixed that, now just very confused why these tests work fine locally and not in CI. Grrr....